app/helpers/events_helper.rb

```diff
@@ -8,10 +8,27 @@ def options_for_events
   def event_link(event)
     return if @event.publisher_url.blank?
 
-    link_to(
-      'Visit the webpage for this event',
-      event.publisher_url,
-      class: 'btn btn-primary'
-    ).html_safe
+    link_to('Visit the webpage for this event',
+            event.publisher_url,
+            class: 'btn btn-primary').html_safe
+  end
+
+  def online_link
```

Comment:
why is this in a helper function? not saying its bad just curious?

Reply:
Im not sure what u mean, the other link helper function is directly above this one, they're in the same place.